app/routes.py

```diff
@@ -1 +1,217 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request, make_response, abort
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goal_bp", __name__, url_prefix="/goals")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+
+@task_bp.route("", methods=["POST"])
+def handle_tasks():
+    request_body = request.get_json()
+    try:
+        new_task = Task(title=request_body["title"],
+                    description=request_body["description"])
```

Comment:
Consider pulling this logic into class method for Task, like how we demo'd in roundtables. If you implemented the from_dict() class method, then this route would just have
```
new_task = Task.from_dict(request_body)
``` 

The responsibility of turning a request body into a Task, should lie with the Task class, not necessarily within the method that handles a POST req.